### DISCUSSION TITLE
Default Config subcleaner.conf location question.

### DISCUSSION THREAD

**Discussion by Healzangels** [KBlixt/subcleaner#59] [Q&A] [answered]
Greetings! 

Apologies for what is a rather basic question but was hoping to confirm. 
The subcleaner.conf I saw in the instructions was mentioned to "copy the default config into the script root directory."

Does this mean you would want to copy subcleaner.conf into the root where the subcleaner.py lives or leaving it in default_config directory and having that within the root folder where subcleaner.py lives.

Thanks for the info and cheers!

**Comment by KBlixt** [comment, marked as answer]
> Greetings! 
> 
> Apologies for what is a rather basic question but was hoping to confirm. 
> The subcleaner.conf I saw in the instructions was mentioned to "copy the default config into the script root directory."
> 
> Does this mean you would want to copy subcleaner.conf into the root where the subcleaner.py lives or leaving it in default_config directory and having that within the root folder where subcleaner.py lives.
> 
> Thanks for the info and cheers! 

No problem,

Copy it from the default_config directory. The one existing in the default config is covered by the git repository. 

But just running subcleaner once will do it all for you.
